### PR TITLE
Docs: document the ENTRY_LIMIT environment variable

### PR DESCRIPTION
**Pull request by tyxla**
This PR documents the `ENTRY_LIMIT` env var that is available since we merged #39390.

#### Changes proposed in this Pull Request

* Document the ENTRY_LIMIT environment variable

#### Testing instructions

* Testing not needed, just verify the documentation suggestion makes sense.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -46,6 +46,12 @@ you can choose to build and run specific sections of Calypso using the `SECTION_
 
 For instance, `SECTION_LIMIT=reader,login npm start` would start Calypso and only build the `reader` and `login` sections.
 
+Additionally, in Calypso we use multiple [Webpack entry points](https://webpack.js.org/concepts/entry-points/) for separating concerns and serving smaller bundles to the user at any given time.
+Building a limited number of entry points speeds up the build process, and to allow that, the `ENTRY_LIMIT` environment variable is available to allow building and running only a specific entry point.
+
+For example: `ENTRY_LIMIT=entry-login,entry-main npm start` would start Calypso and only build the login and the main entry points.
+
+To find all the available enty points, you can refer to the `entry` option in Calypso's primary `webpack.config.js` file.
```
Comment (blowery) [inline docs/install.md]: Could we add a line like this to the section section as well? It's currently missing (and has some grammar goofs)

Comment (tyxla) [reply docs/install.md]: Sure 👍 Added in the latest commits. Care to take another look and tell me if I'm missing something? Grammar isn't my strong side 😉

**Review by blowery — APPROVED** [review]

File: docs/install.md (hunk 1 above)
Comment (blowery) [reply docs/install.md]: LGTM!

**Comment by hambai** [comment]
Thank you @tyxla for this PR and thank you @blowery for the p2 post!!! 👏 

May I suggest one small edit as I really believe we should make those environment variables and what they do more visible. That can save a lot of developers a lot of time and battery (and probably some hair).

Let's move the whole section to [docs/development-workflow.md](https://github.com/Automattic/wp-calypso/blob/master/docs/development-workflow.md) and rename it to:

### Optimizing development environment build times ###

And ideally put the section at or near the top of the document?

**Comment by blowery** [comment]
@hambai That's a good idea. Maybe have it both places?

**Comment by tyxla** [comment]
Thanks @blowery  and @hambai  🙌

**Pull request merged**